script/run.sh

```diff
@@ -43,5 +43,11 @@ if [ -n "$HOME" ]; then
     VOLUMES="$VOLUMES -v $HOME:$HOME"
 fi
 
+# Only allocate tty if we detect one
+if [ -t 1 ]; then
+    DOCKER_RUN_OPTIONS="-ti"
+else
+    DOCKER_RUN_OPTIONS="-i"
```

Comment:
This script is only for the alternative install option of running Compose in a container.

There should be no problem running this script from another one without a tty.